doc/cephfs/fs-volumes.rst

```diff
@@ -97,6 +97,43 @@ The CephX IDs authorized to <vol_name> need to be reauthorized to <new_vol_name>
 on-going operations of the clients using these IDs may be disrupted. Mirroring is
 expected to be disabled on the volume.
 
+Fetch the information of a CephFS volume using::
+
+    $ ceph fs volume info <vol_name>
+    {
+        "mon_addrs": [
+            "192.168.1.7:40280"
+        ],
+        "pending_subvolume_deletions": 0,
+        "pools": {
+            "data": [
+                {
+                    "avail": "106.288848896 GiB",
```

Comment:
GiB or GB needs to be fixed in doc as well.

Reply:
The docs aren't necessary entirely consistent, but I think in doc text we tend to GB in the name of familiarity, if not accuracy.  Lots of people aren't aware of the base-2 vs base-10 difference.

In any event, IMHO this should mirror what the command actually outputs.

Reply:
The code converts the command output to display in GiB as per https://github.com/ceph/ceph/pull/45949#discussion_r884781083 , otherwise by default the usage always comes in bytes.As per your comment what we can do is instead of GiB, display in GB..and that will be reflected in both docs as well as code.